js/Control.OSMGeocoder.js

```diff
@@ -19,7 +19,7 @@ L.Control.OSMGeocoder = L.Control.extend({
 			this._map.fitBounds(bounds);
 
 			// IDE - Map have the power to draw a marker
-			RoadMap.addMarkerSearch(this._map.fitBounds(bounds));
+			RoadMap.setRequestedPositionMarker(this._map.fitBounds(bounds));
```

Comment:
Damage is already done. Here it's a simple function renaming.

Can I open an issue an fix it later ? Vendor code is not so much impacted.